src/App.tsx

```diff
@@ -33,7 +33,7 @@ function LessonInteractiveArea(lesson: ILesson) {
   const exercisesListItems = [];
   for (const exercise of lesson.exercises) {
     exercisesListItems.push(
-      <Snippet.Snippet {...exercise.content} />
+      <Snippet.Snippet key={exercise.title} {...exercise.content} />
```

Comment:
I'm a bit confuzzled as to what key does right now. Is this something we're using for sure later on? And if so, is there a better name rather than `key` (maybe `title`)?

Reply:
This is for react, it requests that you add a key for things in lists to help it more efficiently keep track of changes.  You can see this if you don't include a key and look at the console.